ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-routing.j2

```diff
@@ -6,13 +6,13 @@ ip routing
 !
 no ip routing
 {% endif %}
+{% if ip_icmp_redirect is arista.avd.defined(false) %}
```

Comment:
Please add this to eos_cli_config_gen/README.md and README_v4.0.md.

You can add it under the Routing section like you did in this template.

Reply:
My bad, because of the title **Feat**, I thought you were adding feature. Instead you are just reorganizing the code.